test.json

```diff
@@ -1,13 +1,15 @@
 {
   "source": "./package.json",
   "scripts": {
-    "run": "esy '@test' x OniUnitTestRunner",
-    "run-ci": "esy '@test' x OniUnitTestRunnerCI",
+    "run": "esy '@test' x OniUnitTestRunner && esy @test inline",
+    "run-ci": "esy '@test' x OniUnitTestRunnerCI && esy @test inline",
     "core": "esy '@test' x OniUnitTestRunnerCore",
     "cli": "esy '@test' x OniUnitTestRunnerCLI",
     "model": "esy '@test' x OniUnitTestRunnerModel",
     "exthost": "esy '@test' x OniUnitTestRunnerExtHost",
-    "watch": "esy '@test' dune build --watch --root . -j4"
+    "watch": "esy '@test' dune build --watch --root . -j4",
+    "inline": "esy dune runtest",
```

Comment:
Just tested on Windows - unfortunately this results in an 'assembler error'.

We can fix it by running `esy b dune runtest` (running in the build environment - on Windows, the MingW toolchain, which is needed for linking by dune, is only available in the build environment. Otherwise, dune fails trying to link to run the test). Seems to still work on OSX too.

Reply:
Thanks for the heads up! Fixed in https://github.com/onivim/oni2/pull/1822/commits/f8f37945749b97cdaf10395d72bbb42900401dce